jwst/assign_wcs/nirspec.py

```diff
@@ -1657,31 +1657,24 @@ def _is_valid_slit(domain):
     # collimator to GWA
     collimator2gwa = collimator_to_gwa(reference_files, disperser)
 
-    msa = MSAModel(reference_files['msa'])
     col2det = collimator2gwa & Identity(1) | Mapping((3, 0, 1, 2)) | agreq | \
         gwa2det | det2dms
-    for quadrant in range(1, 6):
-        slits_in_quadrant = [s for s in open_slits if s.quadrant == quadrant]
-        if any(slits_in_quadrant):
-            msa_quadrant = getattr(msa, "Q{0}".format(quadrant))
-            msa_model = msa_quadrant.model
-            msa_data = msa_quadrant.data
-            for slit in slits_in_quadrant:
-                slit_id = slit.shutter_id
-                slitdata = msa_data[slit_id]
-                slitdata_model = get_slit_location_model(slitdata)
-                msa_transform = slitdata_model | msa_model
-                msa2det = msa_transform & Identity(1) | col2det
-                bb = compute_bounding_box(msa2det, wrange, slit.ymin, slit.ymax)
-                valid = _is_valid_slit(bb)
-                if not valid:
-                    log.info("Removing slit {0} from the list of open slits because the "
-                             "WCS bounding_box is completely outside the detector.".format(slit.name))
-                    log.debug("Slit bounding_box is {0}".format(bb))
-                    idx = np.nonzero([s.name == slit.name for s in open_slits])[0][0]
-                    open_slits.pop(idx)
 
-    msa.close()
+    slit2msa = slit_to_msa(open_slits,reference_files['msa'])
```

Comment:
Uh oh. Flake8 isn't going to like the fact that there's no whitespace after the comma in the function argument list.

Reply:
Hm, didn't see it fail the style check, but fixed it up.

Reply:
I guess we don't have that particular test activated. I've just gotten so used to it, that I assume it'll fail the check.